charts/redpanda/values.yaml

```diff
@@ -550,7 +550,22 @@ config:
     # tm_sync_timeout_ms: 2000ms                                   # Time to wait state catch up before rejecting a request
     # tm_violation_recovery_policy: crash                          # Describes how to recover from an invariant violation happened on the transaction coordinator level
     # transactional_id_expiration_ms: 10080min                     # Producer ids are expired once this time has elapsed after the last write with the given producer ID
-  tunable: {}
+  tunable:
+    log_segment_size: 134217728                                    # 128 mb
+    log_segment_size_min: 16777216                                 # 16 mb
+    log_segment_size_max: 268435456                                # 256 mb
+    kafka_batch_max_bytes: 1048576                                 # 1 mb
+    topic_partitions_per_shard: 1000
+    compacted_log_segment_size: 67108864                           # 64 mb
+    max_compacted_log_segment_size: 536870912                      # 512 mb
+    kafka_connections_max: 15100
+    kafka_connection_rate_limit: 1000
+    partition_autobalancing_mode: "continuous"
+    cloud_storage_segment_max_upload_interval_sec: 3600            # 3600 sec = 1 hour
+    group_topic_partitions: 16
+    # cloud_storage_enable_remote_read: true                       # cluster wide configuration for read from remote cloud storage
```

Comment:
Commented lines left in by mistake?

Reply:
No, commented lines are left in as documentation.

Reply:
So now we are adding these items, what are the sideaffects to existing and and upgrading previous versions?

Reply:
This comment is pinned to lines 566. It is parameter available in Redpanda from 22.11.X version as far as I remember. If you have license (not enforced yet) and configures tiered storage https://docs.redpanda.com/docs/platform/data-management/remote-read-replicas/#creating-a-topic-with-archival-storage-or-tiered-storage